composer.json

```diff
@@ -9,7 +9,7 @@
     ],
     "require": {
         "sylius/sylius": "^2.0",
-        "stefandoorn/google-tag-manager-plugin": "^2.0-alpha || ^2.0"
+        "stefandoorn/google-tag-manager-plugin": "^2.0"
```

Comment:
@Prometee For now should this be something like `^2.0 || ^2.0@alpha || ^2.0@beta`?

Reply:
I did not check it but yes it should be ok like that